packages/auth/src/components/EmailPasswordSignUpForm/index.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Button, Input } from 'react-rainbow-components';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { showAppMessage, showAppSpinner, hideAppSpinner } from '../../../../app/src/actions';
+import useFirebaseApp from '../../../../firebase-hooks/src/useFirebaseApp';
+
+const EmailPasswordSignUpForm = (props) => {
+    const intl = useIntl();
+    const { className, style } = props;
+    const [name, setName] = useState('');
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
```

Comment:
We should use redux-form which is setup in RainbowFirebaseApp like react-intl, then it should work out of the box.
Redux-form take care of the state of the form and also make the validation through the validate function.
Note that we should add test for the validate function and also add test cases for when errors show up.